.github/workflows/main.yml

```diff
@@ -17,15 +15,16 @@ jobs:
           os: [ubuntu-20.04]
     steps:
       - name: Setup ROS
-        uses: ros-tooling/setup-ros@0.0.23
+        uses: ros-tooling/setup-ros@0.1.4
         with:
           required-ros-distributions: foxy
       - name: Install Iceoryx Dependencies
         run: sudo apt-get update && sudo apt-get install -y cmake libacl1-dev libncurses5-dev pkg-config
       - name: Build & Test
-        uses: ros-tooling/action-ros-ci@0.0.17
+        uses: ros-tooling/action-ros-ci@0.2.0
         with:
-          package-name: rmw_iceoryx_cpp iceoryx_ros2_bridge
+          package-name: rmw_iceoryx_cpp
           source-ros-binary-installation: foxy
           vcs-repo-file-url: |
             https://gist.githubusercontent.com/Karsten1987/1723f219bb5aa5b81355c7eb1477f867/raw/9dc4d877f6fd0f862070516febdbddea28723b5c/iceoryx.repos
+            https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos
```

Comment:
Should this then replace the previous line, as iceoryx is now in ros2.repos?

Reply:
I will follow the workflow in master branch and make it work.